events/source.go

```diff
@@ -432,12 +352,10 @@ func (b *Notifier) BlockHeights(ctx context.Context) (<-chan int64, <-chan error
 	errChan := make(chan error, 1)
 	b.running, b.shutdown = context.WithCancel(ctx)
 
-	// if no start block has been set explicitly fetch the latest block height
+	// if no start block has been set explicitly try to fetch the latest block height
 	if b.start == 0 {
-
-		b.start = -1
 		height, err := b.getLatestBlockHeight()
```

Comment:
This is where the latest block height call gets the block height of the node instead of the network?

Reply:
yes. I think I'll revert this particular change so the behaviour doesn't change